packages/PreviewCard/PreviewCard.jsx

```diff
@@ -83,8 +87,16 @@ const PreviewCard = ({ header, image, body, footer, href, linkComponent, ...rest
               </Text>
             </Box>
           )}
-          <Box vertical={hasHeaderOrFooter ? 3 : undefined}>
+          <Box
+            vertical={hasHeaderOrFooter ? 3 : undefined}
+            between={hasHeaderOrFooter ? 3 : undefined}
+          >
             <P alt={body}>{newBody}</P>
+            {hrefText && (
+              <ChevronLink href={href}>
+                <HrefTextSmall>{hrefText}</HrefTextSmall>
```

Comment:
can we rename hrefText to linkText.

Also, can we not use TDS Paragraph for font sizing? You should be able to wrap ChevronLink in a Paragraph and then ditch HrefTextSmall.

```jsx
<Paragraph>
  <ChevronLink href={href}>{linkText}</ChevronLink>
</Paragraph>

Reply:
ok